src/App.tsx

```diff
@@ -1,42 +1,42 @@
-import React from 'react';
+import { FC, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
 
-export const tabs = [
+import { Tab } from './types/Tab';
+import { Tabs } from './components/Tabs';
+
+export const tabs: Tab[] = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App: React.FC = () => {
-  return (
-    <div className="section">
-      <h1 className="title">
-        Selected tab is Tab 1
-      </h1>
+export const App: FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState<Tab>(tabs[0]);
 
-      <div data-cy="TabsComponent">
-        <div className="tabs is-boxed">
-          <ul>
-            <li className="is-active" data-cy="Tab">
-              <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-            </li>
+  const handleSelectedItem = (tab: Tab) => {
+    setSelectedTabId(tab);
+  };
 
-            <li data-cy="Tab">
-              <a href="#tab-2" data-cy="TabLink">Tab 2</a>
-            </li>
+  const findTab = (id: string): Tab => (
+    tabs.find(tab => tab.id === id) || tabs[0]
+  );
 
-            <li data-cy="Tab">
-              <a href="#tab-3" data-cy="TabLink">Tab 3</a>
-            </li>
-          </ul>
-        </div>
+  if (!findTab) {
+    setSelectedTabId(tabs[0]);
+  }
```

Comment:
You can have this case, bc you set `tabs[0]` here
![image](https://github.com/mate-academy/react_tabs/assets/91411692/80d4888d-7172-4414-82ca-3a742c21ee76)

```suggestion
```